src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -1299,6 +1301,8 @@ namespace ReportSizingManager {
 						} else if ( OASysEqSizing( CurOASysNum ).HeatingAirFlow ) {
 							// Parent object sets heating flow rate
 							AutosizeDes = OASysEqSizing ( CurOASysNum ).HeatingAirVolFlow;
+						} else if ( DataDesicRegCoil ) {
+							AutosizeDes = FinalSysSizing( CurSysNum ).DesMainVolFlow;
```

Comment:
Yes, I like that solution better than this specific case. There is usually a generic case that could be used or added (e.g., OASysEqSizing or UnitarySysEqSizing). These flags need to be reset so other equipment models don't use them. I did this in ReportUnitarySystem but am now worried about the if (  ! SysSizingCalc ) part of that logic and whether multiple parents using these flags would step on each other (i.e., if these are not reset until sizing is completed).